tests/privacy_startup_network.py

```diff
@@ -0,0 +1,129 @@
+#!/usr/bin/env python3
+
+from testUtils import Account
+from testUtils import Utils
+from Cluster import Cluster
+from WalletMgr import WalletMgr
+from Node import BlockType
+from Node import Node
+from Node import ReturnType
+from TestHelper import TestHelper
+
+import decimal
+import re
+import signal
+import json
+import os
+
+###############################################################
+# privacy_startup_network
+#
+# General test for Privacy to verify TLS connections, and slowly adding participants to the security group and verifying
+# how blocks and transactions are sent/not sent.
+#
+###############################################################
+
+Print=Utils.Print
+errorExit=Utils.errorExit
+cmdError=Utils.cmdError
+from core_symbol import CORE_SYMBOL
+
+args = TestHelper.parse_args({"--port","-p","-n","--dump-error-details","--keep-logs","-v","--leave-running","--clean-run"
+                              ,"--sanity-test","--wallet-port"})
+port=args.port
+pnodes=args.p
+relayNodes=pnodes # every pnode paired with a relay node
+apiNodes=2        # minimum number of apiNodes that 
```

Comment:
That?  That what?  Inquiring minds want to know!

Reply:
ok